mfjcs-rest/README.md

```diff
@@ -1,13 +1,27 @@
 # MFJCS
 
-How to start the MFJCS application
----
+## Starting and configuring Solr
 
-1. Run `mvn clean install` to build your application
-1. Start application with `java -jar target/mfjcs-rest-1.0-SNAPSHOT.jar server config.yml`
-1. To check that your application is running enter url `http://localhost:8080`
+$SOLR_HOME/bin/solr start -c -s solr/home
+$SOLR_HOME/bin/solr zk -upconfig -d solr/conf/mfjcs-items -n mfjcs-items -z localhost:9983
+$SOLR_HOME/bin/solr create -c mfjcs-items -n mfjcs-items
```

Comment:
also: would be nice to indicate which commands one should execute only once
and which ones to execute at each restart?